apps/condo/pages/news/index.tsx

```diff
@@ -0,0 +1,297 @@
+                        />,
+                    ]}
+                />
+            </Col>
+        </Row>
+    )
+}
+
+const NewsPageContent = ({ 
+    baseNewsQuery, 
+    filterMetas, 
+    sortableProperties,
+}) => {
+    const intl = useIntl()
+    const SearchPlaceholder = intl.formatMessage({ id: 'filters.FullSearch' })
+    const EmptyListLabel = intl.formatMessage({ id: 'pages.condo.news.index.emptyList.header' })
+    const EmptyListMessage = intl.formatMessage({ id: 'pages.condo.news.index.emptyList.title' })
+    const CreateNews = intl.formatMessage({ id: 'news.createNews' })
+    const ServerErrorMsg = intl.formatMessage({ id: 'ServerError' })
+
+    const [search, changeSearch ] = useSearch<IFilters>()
+    const handleSearchChange = useCallback((e) => {
+        changeSearch(e.target.value)
+    }, [changeSearch])
+
+
+    const router = useRouter()
+    const { filters, sorters } = parseQuery(router.query)
+    const { filtersToWhere, sortersToSortBy } = useQueryMappers(filterMetas, sortableProperties)
+    //TODO(KEKMUS)now sorting by date sorts only by the createdAt field regardless of whether there is sendAt, need to understand what kind of behavior we expect
+    const sortBy =  sortersToSortBy(sorters, NEWS_DEFAULT_SORT_BY) 
+    const searchNewsQuery = useMemo(() => ({ ...baseNewsQuery, ...filtersToWhere(filters) }),
+        [baseNewsQuery, filters, filtersToWhere])
+
+    const {
+        count: newsWithoutFiltersCount,
+        loading: newsWithoutFiltersCountLoading,
+        error,
+    } = NewsItem.useCount({ where: baseNewsQuery })
+    const loading = newsWithoutFiltersCountLoading
+
+    if (loading || error) {
+        const errorToPrint = error ? ServerErrorMsg : null
+        return <LoadingOrErrorPage loading={loading} error={errorToPrint}/>
+    }
+
+    if (newsWithoutFiltersCount === 0) {
+        return (
+            <EmptyListView
+                label={EmptyListLabel}
+                message={EmptyListMessage}
+                createRoute='/news/create'
+                createLabel={CreateNews}
+            />
+        )
+    }
+
+    return (
+        <>
+            <Row gutter={PAGE_ROW_GUTTER} justify='space-between'>
+                <Col span={24}>
+                    <TableFiltersContainer>
+                        <Input
+                            placeholder={SearchPlaceholder}
+                            onChange={handleSearchChange}
+                            value={search}
+                            allowClear
+                            suffix={<Search size='medium' color={colors.gray[7]} />}
+                        />
+                    </TableFiltersContainer>
+                </Col>
+                <NewsTableContainer
+                    searchNewsQuery={searchNewsQuery}
+                    sortBy={sortBy}
+                    filterMetas={filterMetas}
+                    loading={loading}
+                />
+            </Row>
+        </>
```

Comment:
unnecessary fragment
```suggestion
            <Row gutter={PAGE_ROW_GUTTER} justify='space-between'>
                <Col span={24}>
                    <TableFiltersContainer>
                        <Input
                            placeholder={SearchPlaceholder}
                            onChange={handleSearchChange}
                            value={search}
                            allowClear
                            suffix={<Search size='medium' color={colors.gray[7]} />}
                        />
                    </TableFiltersContainer>
                </Col>
                <NewsTableContainer
                    searchNewsQuery={searchNewsQuery}
                    sortBy={sortBy}
                    filterMetas={filterMetas}
                    loading={loading}
                />
            </Row>
```